setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
+exclude = venv/,digits/,plugins/
```

Comment:
That's worth considering. I think I'll just merge this as-is, but consider moving to that approach on the next incremental lint-fixing PR.